packages/shared/lib/contexts/onboarding/helpers/setImportFile.ts

```diff
@@ -1,6 +1,6 @@
 import { importFile, importFilePath } from '../stores'
 
-export function setImportFile(file: unknown, filePath: string): void {
+export function setImportFile(file: ArrayBuffer, filePath: string): void {
     importFile.set(<Buffer>file)
```

Comment:
After looking into it, I think the argument should be `ArrayBuffer | string` and I would remove the type cast on l.4. There are some related comments on `FileImportView`.